packages/cli/src/commands/env/index.ts

```diff
@@ -42,6 +42,13 @@ const help = () => {
 
   ${chalk.dim('Examples:')}
 
+  ${chalk.gray(
+    '–'
+  )} Pull all Development Environment Variables down from the cloud
+
+      ${chalk.cyan(`$ ${getPkgName()} env pull`)}
```

Comment:
```suggestion
      ${chalk.cyan(`$ ${getPkgName()} env pull <file>`)}
```

This follows the pattern of the other examples below